aws-test/tests/aws_elastic_beanstalk_environment/variables.tf

```diff
@@ -135,8 +135,8 @@ resource "aws_elastic_beanstalk_application" "application_test" {
 
 resource "aws_elastic_beanstalk_environment" "named_test_resource" {
   name                   = var.resource_name
-  application            = "${aws_elastic_beanstalk_application.application_test.name}"
-  solution_stack_name    = "64bit Amazon Linux 2 v3.5.4 running Go 1"
+  application            = aws_elastic_beanstalk_application.application_test.name
+  solution_stack_name    = "64bit Amazon Linux 2 v3.6.3 running Go 1"
```

Comment:
Is there any default value that we can use for this configuration?

Reply:
No @misraved there is no default configuration.